frontend/src/tests/import-export-data/diff-data.js

```diff
@@ -0,0 +1,667 @@
+    ddahs: [
+        {
+            assignment: {
+                applicant: {
+                    first_name: "Harry",
+                    last_name: "Potter",
+                    utorid: "potterh",
+                    email: "harry@potter.com",
+                    student_number: "999666999",
+                    phone: "41888888888",
+                },
+                position: {
+                    position_code: "CSC494",
+                    position_title: "Capstone Project",
+                    hours_per_assignment: 80,
+                    start_date: "2020-12-10T00:00:00.000Z",
+                    end_date: "2021-12-10T00:00:00.000Z",
+                    instructors: [
+                        {
+                            first_name: "Henry",
+                            utorid: "smithh",
+                            last_name: "Smith",
+                            email: "OLD@utoronto.ca",
+                        },
+                        {
+                            email: "gordon.smith@utoronto.ca",
+                            first_name: "Gordon",
+                            last_name: "Smith",
+                            utorid: "smithhg",
+                        },
+                    ],
+                    contract_template: {
+                        template_name: "Regular",
+                    },
+                    duties: "",
+                    qualifications: "",
+                },
+                start_date: "2020-12-10T00:00:00.000Z",
+                end_date: "2021-12-10T00:00:00.000Z",
+                contract_override_pdf: null,
+                hours: 80,
+                active_offer_status: null,
+                active_offer_recent_activity_date: null,
+                wage_chunks: [
+                    {
+                        hours: 30,
+                        rate: 50,
+                        start_date: "2020-12-10T00:00:00.000Z",
+                        end_date: "2020-12-31T00:00:00.000Z",
+                    },
+                    {
+                        hours: 30,
+                        rate: 50,
+                        start_date: "2021-01-01T00:00:00.000Z",
+                        end_date: "2021-06-30T00:00:00.000Z",
+                    },
+                    {
+                        hours: 20,
+                        rate: 50,
+                        start_date: "2021-07-01T00:00:00.000Z",
+                        end_date: "2021-12-10T00:00:00.000Z",
+                    },
+                ],
+            },
+            duties: [
+                {
+                    description: "Initial training",
+                    hours: 30,
+                },
+                {
+                    description: "Marking the midterm",
+                    hours: 50,
+                },
+            ],
+        },
+    ],
+};
+
+// modified instructor data used for computing the diff with initial objects
+export const modifiedInstructorData = [
```

Comment:
I thought you were going to make deep copies of the `reduxStoreData` and then modify the parts that you wanted changed...?